knimin/handlers/pm_plate_map.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+from tornado.web import authenticated
+from knimin.handlers.base import BaseHandler
+from knimin import db
+from knimin.handlers.access_decorators import set_access
+from json import dumps
+
+
+@set_access(['Admin'])
+class PMPlateMapHandler(BaseHandler):
+    @authenticated
+    def get(self):
```

Comment:
Makes sense. I will separate them into two handlers, one for HTML, one for JSON. In the future there will be more AJAX calls per page (see below). It is probably better to group them into one handler.